components/api_request_helper/api_request_helper.cc

```diff
@@ -63,6 +59,7 @@ void ParseJsonUsingRust(
             }
           },
           std::move(callback)));
+  return;
```

Comment:
`return` is not necessary here.

Reply:
Fixed by @cdesouza-chromium here https://github.com/brave/brave-core/pull/26068/commits/dd4b1184ec0da83e7e7afbd05c1ca0b649e21638.